drivers/bestfortress.c

```diff
@@ -500,14 +602,25 @@ void upsdrv_initups(void)
 	}
 
 	upsfd = ser_open(device_path);
+	if (INVALID_FD(upsfd)) {
+		upslogx(LOG_WARNING, "%s: failed to open %s",
+			__func__, device_path);
+		/* \todo: Deal with the failure */
+	}
+
+	/* ser_set_speed returns int 0 always; fatal if ioctl fails */
 	ser_set_speed(upsfd, device_path, speed);
-	/* TODO: probe ups type */
 
-	/* the upsh handlers can't be done here, as they get initialized
-	 * shortly after upsdrv_initups returns to main.
-	 */
```

Comment:
Should these comments have disappeared? Do they no longer apply?

Reply:
I removed them on purpose because I don't think they make sense.   The docs define what each function is supposed to do and explaining that we don't do something not documented doesn't seem sensible to me.  These things are set in initinfo which seems like the right place.   I don't want to add comments for why we aren't setting everything else that is set in initinfo.  Is that ok?

Reply:
Sounds reasonable from this PoV, thanks :)